Add instruction for adding launch image for iOS release (#7936)
flutter/website: src/deployment/ios.md

1. Verify the icon has been replaced by running your app using
   `flutter run`.

## Add a launch image

Similar to the app icon, you may also want to replace the placeholder launch image: 

1. In the Xcode project navigator, select `Assets.xcassets` in the
   `Runner` folder. Update the placeholder launch image with your own launch image. 

> 1. Verify the launch image has been replaced by running your app using `flutter run`.

Reviewer: **suggestion:** Flip the sentence to remove passive voice.
```suggestion
1. Run  `flutter run` to verify that the launch image replacement.
```

Reviewer 2: @atsansone, your suggestion was grammatically wrong. I've tweaked:

```suggestion
1. Verify the new launch image by hot restarting your app. (Don't use `hot reload`.)
```
